src/tests/topp/ConvertTSVToTraML_2_output.TraML

```diff
@@ -81,6 +81,11 @@
       <cvParam cvRef="MS" accession="MS:1001226" name="product ion intensity" value="1"/>
       <cvParam cvRef="MS" accession="MS:1002007" name="target SRM transition"/>
       <userParam name="annotation" type="xsd:string" value="y5"/>
+      <userParam name="detecting_transition" type="xsd:integer" value="1"/>
```

Comment:
The default behavior for CSV and TraML is now to explicitly write the additional annotation fields. In case these fields are missing, the previous defaults are used and written. We could also make this optional, but I'm not sure whether this will make the handling for users simpler.